content/ui-patterns/saving.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Saving
+---
+
+GitHub's save patterns help developers to store and update their configurations for different features throughout GitHub's various interfaces. These changes should be represented in the UI accurately, quickly, and obviously. Their behavior should be easy to reason about and inspire confidence and trust.
+
+All configuration flows should be punctuated with CTAs that feature a logical term such as "Create", "Save", "Delete", or "Update" (e.g. "Create" a new repository, "Add" an SSH key to your profile, "Save" security preferences, "Update" a repository's description, "Delete" an old email address). 
```

Comment:
Are these 2nd and 3rd paragraph the TLDR of the rest of the doc? I'm thinking if we should go into detail/guidelines in the intro area, or leave it more "intro-y". I can explain better if needed 😄